mapbox_tilesets/scripts/cli.py

```diff
@@ -119,10 +119,19 @@ def publish(tileset, token=None, indent=None):
     )
     r = s.post(url)
     if r.status_code == 200:
-        click.echo(json.dumps(r.json(), indent=indent))
+        response_msg = r.json()
+        click.echo(json.dumps(response_msg, indent=indent))
+
+        studio_url = click.style(
+            f"https://studio.mapbox.com/tilesets/{tileset}", bold=True
+        )
+        job_id = response_msg["jobId"]
+        job_cmd = click.style(f"tilesets job {tileset} {job_id}", bold=True)
+        message = f"\n✔ Successfully published. Visit {studio_url} or run {job_cmd} to view the status of your tileset."
```

Comment:
Tangential to the substance of the change, but should we really say "successfully published" here when we don't know yet whether the actual publication will succeed or fail? Could it be "publication started" or something like that instead?

Reply:
Good call out @ericfischer - I want to refrain from adding more verbiage to an already verb-heavy endpoint. My perspective says that the request to publish this tileset was successful. Plus the sentence suggesting how to view the "status" of your tileset reads as if it's not guaranteed the publish will result in a complete tileset. 

I do like the idea of saying something like:

```
✔ Tileset job recieved. <...> 
```

What do you think?

Reply:
Sounds great! Thanks for changing it.